website/docs/r/l3_ext_subnet.html.markdown

```diff
@@ -36,36 +36,32 @@ Manages ACI l3 extension subnet
 		tn_rtctrl_profile_dn  = aci_route_control_profile.bgp_route_control_profile.id
 		direction = "import"
 	  }
+	  relation_l3ext_rs_subnet_to_profile {
+		tn_rtctrl_profile_dn  = aci_route_control_profile.bgp_route_control_profile_2.id
+		direction = "export"
+	  }
 	}
 
 ```
 
-## Argument Reference
-
 ## Argument Reference ##
-* `external_network_instance_profile_dn` - (Required) Distinguished name of parent ExternalNetworkInstanceProfile object.
-* `ip` - (Required) ip of Object l3 extension subnet.
-* `aggregate` - (Optional) Aggregate Routes for l3 extension subnet. Allowed values are "import-rtctrl", "export-rtctrl","shared-rtctrl" and "none".
-* `annotation` - (Optional) annotation for object l3 extension subnet.
-* `name_alias` - (Optional) name_alias for object l3 extension subnet.
-* `scope` - (Optional) The list of domain applicable to the capability. Allowed values are "import-rtctrl", "export-rtctrl", "import-security", "shared-security" and "shared-rtctrl". Default is "import-security".
 
-* `relation_l3ext_rs_subnet_to_profile` - (Optional) Relation to Route Control Profile (class rtctrlProfile). Cardinality - N_TO_ONE. Type - Set of Map.
-	* `relation_l3ext_rs_subnet_to_profile.tn_rtctrl_profile_name` - **Deprecated** (Required if tn_rtctrl_profile_dn is not used)(Optional) Associates the external EPGs with the route control profiles.
-	* `relation_l3ext_rs_subnet_to_profile.tn_rtctrl_profile_dn` - (Optional) Associates the external EPGs with the route control profiles.
-	* `relation_l3ext_rs_subnet_to_profile.direction` - (Required) Relation to configure route map for each BGP peer in the inbound and outbound directions.
-
-* `relation_l3ext_rs_subnet_to_profile` - (Optional) Relation to class rtctrlProfile. Cardinality - N_TO_M. Type - Set of Map.
-* `relation_l3ext_rs_subnet_to_rt_summ` - (Optional) Relation to class rtsumARtSummPol. Cardinality - N_TO_ONE. Type - String.
-
-## Attribute Reference
-
-The only attribute that this resource exports is the `id`, which is set to the
-Dn of the l3 extension subnet.
+* `external_network_instance_profile_dn` - (Required) Distinguished name of the parent External Network Instance Profile object.
+* `ip` - (Required) ip of the L3 Extension Subnet object.
```

Comment:
of the External EPG Subnet
Update all other references too.

Reply:
ok